custom_dc/custom_data.md

```diff
@@ -116,11 +180,214 @@ geoId/06,2021,555,666
 geoId/08,2021,10,10
 ```
 
-## Write the data config file
+### Write the JSON config file
+
+You must define a `config.json` in the top-level directory where your CSV files are located. With the implicit schema method, you need to provide 3 specifications:
+- The input files location and entity type
+- The sources and provenances of the data
+- Optionally, additional properties of the statistical variables you've used in the CSV files
+
+Here is an example of how the config file would look for WHO CSV file we defined earlier. More details are below.
+
+```json
+{
+  "inputFiles": {
```

Comment:
It may also be worth mentioning a new feature added in https://github.com/datacommonsorg/import/pull/352

 inputFiles can now point at csv files in subdirectories like:

```
{
  "inputFiles": {
    "data/*.csv": { ... }
  },
  "includeInputSubdirs": true
  ...
}
```

Previously, the csv files had to be in the same directory as the config.json. 

You'll need to set `"includeInputSubdirs": true` for this to work

Reply:
+1 for documenting this (can be in a separate PR)

Also, it applies to all files (csv, mcf)

Reply:
Ah, thank you very much for this!

Reply:
@ajaits This must be what one.org missed. :-)  I forgot there was a required option to set this. So I need to rewrite some other stuff above to mention MCF as well (I thought it needed to be at the top level with the JSON.)

Reply:
OK PTAL.